static/app/components/modals/inviteMembersModal/index.spec.tsx

```diff
@@ -1,3 +1,4 @@
+import {ComponentProps} from 'react';
```

Comment:
nit: no need to import, can just use `React.ComponentProps` we don't have a lint or anything for this though so 🤷

Reply:
What's your logaf for this one?

Reply:
we'll fx in lint at some point, was more of just a heads up